tests/scripts/pulpcore/test_orphans.sh

```diff
@@ -6,3 +6,40 @@
 expect_succ pulp orphan cleanup
 
 test "$(echo "${OUTPUT}" | jq -r '.state' )" = "completed"
+
+pulp debug has-plugin --name "file" || exit 3
+
+if [ "$(pulp debug has-plugin --name "core" --min-version "3.14.0")" = "true" ]
```

Comment:
```suggestion
if pulp debug has-plugin --name "core" --min-version "3.14.0"
```
This command is meant to be used as a condition.